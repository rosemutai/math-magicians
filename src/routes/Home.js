import React from 'react';

const Home = () => (
  <div className="w-full mt-8 px-12">
    <h2 className="title text-xl font-bold my-3">Welcome to our page</h2>
    <div className="w-1/2 leading-7">
      <p>
        Math magicians is a website for all fans of mathematics.
        It is a Single Page App (SPA) that allows users to Make simple calculations
        by using our simple calculator and view a random quote on our quotes page.
      </p>
    </div>
  </div>
);

export default Home;
