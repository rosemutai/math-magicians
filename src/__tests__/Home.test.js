import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Home from '../routes/Home';

describe('Home', () => {
  it('renders HomePage correctly', () => {
    const { container } = render(
      <Router>
        <Home />
      </Router>,
    );
    expect(container).toMatchSnapshot();
  });
});
