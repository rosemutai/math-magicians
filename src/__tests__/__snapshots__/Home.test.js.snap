// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders HomePage components 1`] = `
<div>
  <div
    class="w-full mt-8 px-12"
  >
    <h2
      class="title text-xl font-bold my-3"
    >
      Welcome to our page
    </h2>
    <div
      class="w-1/2 leading-7"
    >
      <p>
        Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to Make simple calculations by using our simple calculator and view a random quote on our quotes page.
      </p>
    </div>
  </div>
</div>
`;

exports[`Home renders HomePage correctly 1`] = `
<div>
  <div
    class="w-full mt-8 px-12"
  >
    <h2
      class="title text-xl font-bold my-3"
    >
      Welcome to our page
    </h2>
    <div
      class="w-1/2 leading-7"
    >
      <p>
        Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to Make simple calculations by using our simple calculator and view a random quote on our quotes page.
      </p>
    </div>
  </div>
</div>
`;
